licenses/views.py

```diff
@@ -168,6 +174,61 @@ def normalize_path_and_lang(request_path, jurisdiction, language_code):
     return request_path, language_code
 
 
+def view_deed(
+    request,
+    unit,
+    version,
+    category=None,
+    jurisdiction=None,
+    language_code=None,
+):
+    request.path, language_code = normalize_path_and_lang(
+        request.path, jurisdiction, language_code
+    )
+    path_start = os.path.dirname(request.path)
+    legalcode = get_object_or_404(
+        LegalCode,
+        deed_url=request.path,
+    )
+    license_rel_path = os.path.relpath(legalcode.license_url, path_start)
+
+    license = legalcode.license
+    category, category_title = get_category_and_category_title(
+        category,
+        license,
+    )
+    language_code = legalcode.language_code  # CC language code
+    languages_and_links = get_languages_and_links_for_legalcodes(
+        path_start, license.legal_codes.all(), language_code, "deed"
+    )
+
+    if license.unit == "CC0":
+        body_template = "includes/deed_cc0_body.html"
+    elif license.unit in UNITS_LICENSES and license.version == "4.0":
+        body_template = "includes/deed_40_body.html"
+    else:
+        # Default to 4.0 - or maybe we should just fail?
+        body_template = "includes/deed_40_body.html"
```

Comment:
The outcome of these cases isn't substantively different. As the comment suggests, perhaps failure is an option?